frontend-zs/src/components/stories/StoryList.component.js

```diff
@@ -0,0 +1,116 @@
+                        <Media>
+                            <Image
+                                className="mr-1"
+                                src={likeIcon}
+                                width="23"
+                                height="23"
+                                fluid
+                            />
+                            <Media.Body>
+                                <h5><strong>{props.story.numberOfLikes.length}</strong></h5>
+                            </Media.Body>
+                        </Media>
+                    </Row>
+                </Container>
+            </Card.Body>
+        </Card>
+    </Col>
+)
+
+export default class StoryList extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.onChangeSearchCriteria = this.onChangeSearchCriteria.bind(this)
+
+        /* TODO: exchange hard coded data with data from the database */
+        this.state = {
+            searchCriteria: '',
+            stories: [{
+                _id: 1,
+                title: 'Story title - 1',
+                author: 'Author',
+                shortDescription: 'Some quick example text to build on the card title and make up the bulk of the cards content.',
+                category: 'category1',
+                numberOfLikes: ['id1']
+            },
+            {
+                _id: 2,
+                title: 'Story title - 2',
+                author: 'Author',
+                shortDescription: 'Some quick example text to build on the card title and make up the bulk of the cards content.',
+                category: 'category2',
+                numberOfLikes: ['id1', 'id2']
+            },
+            {
+                _id: 3,
+                title: 'Story title - 3',
+                author: 'Author',
+                shortDescription: 'Some quick example text to build on the card title and make up the bulk of the cards content.',
+                category: 'category3',
+                numberOfLikes: ['id1', 'id2', 'id3']
+            }]
+        }
+    }
+
+    onChangeSearchCriteria(e) {
+        this.setState({
+            searchCriteria: e.target.value
+        })
+    }
+
+    renderStoryList() {
+        let criteria = this.state.searchCriteria
+        let filteredsStories = this.state.stories.filter(function (story) {
+            return story.title.includes(criteria)
+        })
+
+        return filteredsStories.map(function (currentStory, index) {
+            return <StoryCard story={currentStory} key={index} />
+        })
+    }
+
+    render() {
+        return (
+            <Container className="zs-style">
+                <br />
+                <Form.Control
+                    type="text"
+                    placeholder="Suche nach Titel"
+                    aria-label="Search"
+                    value={this.state.searchCriteria}
+                    onChange={this.onChangeSearchCriteria}
+                />
+                <br />
```

Comment:
I think we shouldn't be using ``<br />`` instead of margins, what do you think?
https://stackoverflow.com/questions/12788503/style-wise-br-or-padding-margin-elements